controls/SV-235148.rb

```diff
@@ -69,10 +69,13 @@
   SELECT @@ssl_fips_mode;
   )
 
-  ssl_params = sql_session.query(query_ssl_params).results
-
+describe.one do
+  describe '@@ssl_fips_mode' do
+    subject { ssl_params.column('@@ssl_fips_mode').join }
+    it { should cmp 'ON' }
+  end
   describe '@@ssl_fips_mode' do
     subject { ssl_params.column('@@ssl_fips_mode').join }
-    it { should match /ON|STRICT/ }
```

Comment:
`end` for `describe.one do` is missing

Reply:
The end is already there for the describe.one

Reply:
Discussed this later on a call and talked about using `mock://` to check for syntax errors.